docs/visual-basic/misc/file-io-of-a-structure-with-field-fieldname-of-type-typename-is-not-valid.md

```diff
@@ -20,5 +20,5 @@ The structure you are attempting to write to a file contains an unsupported type
 -   Check the structure to determine where the unsupported type occurs, and then change it to a valid type.  
   
 ## See Also  
- [My.Computer.FileSystem.WriteAllBytes Method](http://msdn.microsoft.com/en-us/b1a24dc1-eac8-4e22-8ffa-cc3bacbaf826)  
+ [My.Computer.FileSystem.WriteAllBytes](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.WriteAllBytes)  
```

Comment:
This link, which also appears in the last two topics in the build report, is broken. It should end in %2A.